src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -101,6 +115,16 @@
                         .build());
     }
 
+    private void cleanupCaseDataBeforeSubmittingToCcd(CaseDetails caseDetails) {
```

Comment:
Similarly to the methods in the previous class, I think this should return a modified copy of the case data. It will help readability, in my opinion.